examples/MvcWeb/Services/ProjectService.cs

```diff
@@ -8,30 +12,69 @@ namespace MvcWeb.Services
     public interface IProjectService
     {
         Task<bool> TryAddProject(ProjectViewModel projectViewModel);
+
+        Task ChangeStatus(int id, ProjectStatus rejected);
+
+        IEnumerable<Project> GetAll();
+
+        Project GetbyId(int id);
     }
 
     public class ProjectService : IProjectService
     {
-        public ProjectService(IUnitOfWork unitOfWork)
+        public ProjectService(IUnitOfWork uow)
         {
-            UnitOfWork = unitOfWork;
+            _uow = uow;
         }
 
-        private IUnitOfWork UnitOfWork { get; }
+        private readonly IUnitOfWork _uow;
 
         public async Task<bool> TryAddProject(ProjectViewModel projectViewModel)
```

Comment:
sorry, I know this is not yours - but it should not return bool, please change it to void and throw exception in case something went wrong